build/lint.cloudbuild.yaml

```diff
@@ -16,9 +16,11 @@ steps:
 - name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   id: 'lint'
   args: ['/usr/local/bin/test_lint.sh']
+  env:
+  - "EXCLUDE_LINT_DIRS=\\./modules/view_generator/scripts"
 tags:
 - 'ci'
 - 'lint'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0.6.0'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '0'
```

Comment:
Why was the version removed here?

Reply:
This is to pin to a major version instead of the minor version, reducing churn.